addons/account/wizard/account_invoice_refund.py

```diff
@@ -134,36 +134,33 @@ def invoice_refund(self):
         return self.compute_refund(data_refund)
 
     @api.multi
-    def compute_charge_note(self):
-        invoice = self.env['account.invoice']
+    def compute_charge_note(self, form, inv):
```

Comment:
@umiphos 

In a stable version, you cannot change a method sign :(

Reply:
What about 
```python
compute_charge_note(self, form=False, inv=False):
```